charts/hedera-json-rpc-relay-websocket/templates/configmap.yaml

```diff
@@ -13,7 +13,7 @@ data:
   MIRROR_NODE_RETRIES: {{ .Values.config.MIRROR_NODE_RETRIES | quote }}
   MIRROR_NODE_RETRY_DELAY: {{ .Values.config.MIRROR_NODE_RETRY_DELAY | quote }}
   MIRROR_NODE_URL: {{ .Values.config.MIRROR_NODE_URL | quote }}
-  SUBSCRIPTIONS_ENABLED: "true"
+  SUBSCRIPTIONS_ENABLED: {{ .Values.config.SUBSCRIPTIONS_ENABLED | quote }}
```

Comment:
This doesn't need to be configurable. If it is anything other than `true` the subscriptions will not work. It's only required for tihs chart.